bitsail-connectors/connector-cdc/pom.xml

```diff
@@ -33,13 +33,17 @@
     </modules>
 
     <properties>
-        <maven.compiler.source>8</maven.compiler.source>
-        <maven.compiler.target>8</maven.compiler.target>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <debezium.version>1.6.4.Final</debezium.version>
     </properties>
 
-    <dependencies>
-
-    </dependencies>
 
+    <dependencyManagement>
```

Comment:
some cdc connector may not use debezium in the future, let's keep this in the submodule.